types/spectest/tests/consensusdata/proposer/versioned_blinded_block_consensus_data_nil.go

```diff
@@ -0,0 +1,31 @@
+package consensusdataproposer
+
+import (
+	"github.com/attestantio/go-eth2-client/spec"
+
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// VersionedBlindedBlockConsensusDataNil tests an invalid consensus data with bellatrix block
+func VersionedBlindedBlockConsensusDataNil() *ProposerSpecTest {
+	cd := &types.ConsensusData{
+		Duty:    testingutils.TestingProposerDuty,
+		Version: spec.DataVersionBellatrix,
+		DataSSZ: nil,
+	}
+
+	cdSSZ, err := cd.MarshalSSZ()
+	if err != nil {
+		panic(err.Error())
```

Comment:
so basically this will always fail here?

Reply:
no, marshaling ssz with nil bytes is possible